### DISCUSSION TITLE
Hashing Tutorial Using Feature Barcoding 10x Cellranger Count Pipeline

### DISCUSSION THREAD

**Discussion by erichpeterson** [satijalab/seurat#4022] [Q&A] [answered]
Hi,
We are trying to perform an analysis on three mice samples, in which each of three mice was tagged with a unique antibody, and then the samples were pooled and sequenced. A gene expression and a "feature barcoding" experiment was run per 10x guidelines. At the time of running, we didn't know that the 10x Cellranger _count_ does not support hashing. 

We are trying to get the Seurat tools to work with this data and using the vignette found here as a template: https://satijalab.org/seurat/archive/v3.1/hashing_vignette.html

We were just wanting to see if the steps we are taking below seem appropriate given the situation:
1. Ran the samples via the Cellranger _mkfastq_ as normal
2. Ran Cellranger count using the "feature barcoding" method found here: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis (providing library CSV with the fastq files for the gene expression and feature barcoding samples, as well as the feature reference CSV).
3. The output directory "filtered_feature_bc_matrix" containing the _barcodes.tsv.gz_, _features.tsv.gz_, and _matrix.mtx.gz_ were used as input for the tutorial at https://satijalab.org/seurat/archive/v3.1/hashing_vignette.html 
4. The vignette was run as-is, except for one notable change, which is the crux of our question: it appears as though after running _Read10X()_ the gene expression data is placed in the first element of a list and the feature barcoding portion is placed in the second. So, we edited the vignette to look something like this:
```r
htos <- Read10X("./HTO")
umis <- htos[[1]]
htos <- htos[[2]]
```

Is this the way it should be done?

Thanks

**Comment by timoast** [comment, marked as answer]
Yes that's correct
